### DISCUSSION TITLE
CTS 600  in VP 18c

### DISCUSSION THREAD

**Discussion by Mi2-cz** [veista/nilan#49] [Q&A]
Hi I was surprised that I have this board.
I spend a lot of time to connect to this board via IP-MODBUS device but nothing. 
I have this SW. 
![20230228_213010](https://user-images.githubusercontent.com/62652695/221986904-9d9221e3-69fb-4f3a-bc41-d5bdde5f513f.jpg)
![20230228_213014](https://user-images.githubusercontent.com/62652695/221987022-fb876167-382c-4771-be05-fc3367a6f48e.jpg)

Can you help me

**Comment by veista** [comment]
Hi. Unfotunately the cts600 is not supported. You would have to get a cts602 upgrade from Nilan.

**Reply by Mi2-cz** [reply]
Thank you for confirm so I have to try something different. I saw that someone was able to connect to CTS 600 via mqtt from arduino. I know that Nilan has upgrade kit but is quite expensive.

**Reply by veista** [reply]
If you get it to work, please tell. I can then try adding it to the integration.

**Reply by Mi2-cz** [reply]
Sure, I think that i have to buy different (proper) device. My 485 to TCP/IP device do not have GND input - so maybe that is the reason why i can't connect to CTS600

**Reply by veista** [reply]
I quickly googled the cts600. It is possible to replace the HMI panel with remote connection, but this itegration can not emulate the button presses, so keep in mind that most if not all settings will be inaccessible remotely with firmware <2.38. Best recourse in my opinion is to look for a used or new cts602 board and get the latest update from Nilan.

**Reply by veista** [reply]
Or then you could make hardware that is capable to emulate the panel fully and also gives access to the panel remotely.

**Reply by Mi2-cz** [reply]
Hi Veista
I found maybe something useful but my knowledge is poor for understand it :-(
https://www.ihc-user.dk/forum/forums/topic/2589-ihc-m-bus-nilan-og-dantherm-ventilation/
[Prot485.pdf](https://github.com/veista/nilan/files/10901781/Prot485.pdf)
[dos.txt](https://github.com/veista/nilan/files/10901782/dos.txt)

**Reply by veista** [reply]
Yes. You would have to make a device that replaces the HMI panel to make it work. Take a look https://github.com/jascdk/Nilan_Homeassistant

You would need something like that to replace the panel.
